COPYING

```diff
@@ -1,8 +1,22 @@
 Copyright (c) 2015 Sergi Granell
 Copyright (c) 2015 Danielle Church
+Copyright (C) 2016 Yifan Lu
+Copyright (C) 2020 Asakura Reiko
```

Comment:
could you add just `vitasdk`? :)

Reply:
People can add their names if they want or not add their names if they dont want. Dont see the problem here.

Reply:
might be. but, I just thought `vitasdk` is more community friendly naming.

https://github.com/vitasdk/vita-headers/blob/master/LICENSE.md
https://github.com/vitasdk/vita-headers/issues/104